### PR TITLE
Abstract class in C++

### LINKED ISSUE #7073: Abstract class in C++
### Description

This documentation will explain about the abstract class in c++ 

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by I-ArchanaDash**
<hr>

## Description 📜

Added documentation on Abstract class in c++ .
Fixes #7073

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/OOPs in C++/Abstract class in C++/Abstract_class.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Abstract Class in C++
+▶️ Abstract class is a class whose instantiation is restricted.
```
Comment (deeqakkk) [inline C_CPP/OOPs in C++/Abstract class in C++/Abstract_class.md:2]: Using such types of colored emojis is not preferred, remove them and use simple bullet points.

**Review by deeqakkk — CHANGES_REQUESTED** [review]
And also update the readme file with its directory.

File: C_CPP/OOPs in C++/Abstract class in C++/Abstract_class.md (hunk 1 above)
Comment (I-ArchanaDash) [reply C_CPP/OOPs in C++/Abstract class in C++/Abstract_class.md]: @deeqakkk Updated the required changes.

**Review by deeqakkk — APPROVED** [review]
@Saviour1001 Content is good. Please verify and merge.

**Pull request merged**